packages/commonwealth/client/scripts/hooks/useAppStatus.ts

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const useAppStatus = () => {
+  const [isAddedToHomeScreen, setIsAddedToHomeScreen] = React.useState(true);
+  const [isStandalone, setIsStandalone] = React.useState(true);
+  const [isMarketingPage, setIsMarketingPage] = React.useState(false);
+  const [isIOS, setIsIOS] = React.useState(false);
+  const [isAndroid, setIsAndroid] = React.useState(false);
+
+  React.useEffect(() => {
+    setIsStandalone(
+      process.env.NEXT_PUBLIC_ONLY_STANDALONE === 'true' ? true : false,
+    );
+    setIsAddedToHomeScreen(
+      window.matchMedia('(display-mode: standalone)').matches,
+    );
+    setIsIOS(
+      window.navigator.userAgent.match(/(iPad|iPhone|iPod)/g) ? true : false,
+    );
+    setIsAndroid(window.navigator.userAgent.match(/Android/g) ? true : false);
+    setIsMarketingPage(window.location.pathname === '/');
+  }, []);
```

Comment:
this effect is not necessary. Only thing it does is setting initial values, you can do this right away in state initalization. However, if those values are not mutable, why do we even need state? cant we just return those values like 
```
{
isAndroid: !!window.navigator.userAgent.match(/Android/g)
...
} 
```